src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAOSpec.scala

```diff
@@ -38,6 +39,63 @@ class GraphWorkspaceDAOSpec extends FlatSpec with Matchers with OrientDbTestFixt
     }
   }
 
+  it should "save a new workspace with a realm" in withDefaultTestDatabase { dataSource =>
+    dataSource.inTransaction() { txn =>
+      val user = RawlsUser(RawlsUserSubjectId("user@secret.com"), RawlsUserEmail("user@secret.com"))
+      authDAO.saveUser(user, txn)
+      val secretGroup = makeRawlsGroup("SecretRealm", Set(user), Set.empty)
+      authDAO.saveGroup(secretGroup, txn)
+      val secretGroupRef = RawlsGroupRef(groupName = secretGroup.groupName)
+      val secretWorkspace = Workspace(
+        namespace = testData.wsName.namespace,
+        name = testData.wsName.name,
+        realm = Some(secretGroupRef),
+        workspaceId = "aWorkspaceId",
+        bucketName = "aBucket",
+        createdDate = testDate,
+        lastModified = testDate,
+        createdBy = "Secret User",
+        attributes = Map("workspace_attrib" -> AttributeString("foo")),
+        accessLevels = Map.empty
+      )
+      workspaceDAO.save(secretWorkspace, txn)
+      val savedWorkspace = workspaceDAO.findById(secretWorkspace.workspaceId, txn).get.workspace
+      assert {
```

Comment:
I'll test this out. I noticed that the lastModified date is not exactly the same date as what was passed in - there is a very slight difference between the two or there is a post-insert modification of some kind affecting it.